src/shared/scrapers/JHU-USA.js

```diff
@@ -0,0 +1,136 @@
+import fipsCodes from 'country-levels/fips.json';
+import * as fetch from '../lib/fetch/index.js';
+import * as parse from '../lib/parse.js';
+import * as geography from '../lib/geography/index.js';
+import * as transform from '../lib/transform.js';
+import datetime from '../lib/datetime/index.js';
+import maintainers from '../lib/maintainers.js';
+
+// Set county to this if you only have state data, but this isn't the entire state
+const UNASSIGNED = '(unassigned)';
+
+function findCountryLevelID(fips) {
```

Comment:
is this too simple to be useful as an abstraction layer? why not just call fipsCodes[fips]?

Reply:
I agree!